{{cookiecutter.project_slug}}/.github/workflows/publish.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       run: |
         python setup.py sdist bdist_wheel
     - name: Publish to PyPI
-      uses: pypa/gh-action-pypi-publish@v1
+      uses: pypa/gh-action-pypi-publish@master
```

Comment:
I dunno. It seems like sticking with versioned releases is a more stable solution here. From the readme of this action even:

> Pro tip: instead of using branch pointers, like master, pin versions of Actions that you use to tagged versions or sha1 commit identifiers. This will make your workflows more secure and better reproducible, saving you from sudden and unpleasant surprises

Reply:
(At [https://github.com/pypa/gh-action-pypi-publish](https://github.com/pypa/gh-action-pypi-publish))

Reply:
Huh, ok, as in convo there is no non-alpha release for this action so master is the option then ¯\_(ツ)_/¯